app/src/main/java/io/reark/rxgithubapp/advanced/data/stores/UserSettingsStore.java

```diff
@@ -50,7 +50,7 @@ public UserSettingsStore(final int userId, @NonNull final ContentResolver conten
 
     private void initUserSettings(final int userId) {
         getOnce(userId)
-                .toObservable()
+                .toMaybe()
```

Comment:
Is this needed? Removing it should work as well

Reply:
Indeed, removed